builder/bundles.go

```diff
@@ -475,12 +493,20 @@ func genUpdateBundleSpecialFiles(chrootDir string, cfg *buildBundlesConfig, b *B
 }
 
 func installBundleToFull(packagerCmd []string, buildVersionDir string, bundle *bundle) error {
+	var err error
 	baseDir := filepath.Join(buildVersionDir, "full")
 	args := merge(packagerCmd, "--installroot="+baseDir, "install")
-	args = append(args, bundle.AllPackages...)
-	err := helpers.RunCommandSilent(args[0], args[1:]...)
-	if err != nil {
-		return err
+	if len(bundle.DirectPackages) > 0 {
+		// There were packages directly included for this bundle so
+		// install to full chroot. This check is necessary so we don't
+		// call dnf install with no package listed.
+		for p := range bundle.DirectPackages {
+			args = append(args, p)
```

Comment:
Why does this need to be in a loop, instead of just `args = append(args, bundle.DirectPackages...)` like above?

Reply:
Because this is a map not a slice.

Reply:
Oh. Yes. Very well, then.